src/yunohost/tools.py

```diff
@@ -189,17 +215,21 @@ def tools_maindomain(auth, new_domain=None):
         else:
             logger.info(out)
 
-    # Generate SSOwat configuration file
-    app_ssowatconf(auth)
 
-    # Regen configurations
-    try:
-        with open('/etc/yunohost/installed', 'r') as f:
-            service_regen_conf()
-    except IOError:
-        pass
+def _is_inside_container():
+    """
+    Check if we're inside a container (i.e. LXC)
 
-    logger.success(m18n.n('maindomain_changed'))
+    Returns True or False
+    """
+
+    p = subprocess.Popen("sudo grep -qa container=lxc /proc/1/environ".split(),
```

Comment:
>yunohost can run in docker today

Well, in the last discussions I saw about docker, Yunohost was dropping support of docker...

But I implemented the solution in d67b391 anyway.